CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 1.0.0 (IN PROGRESS)
 
-* Display patron details on patron's barcode Scan. Ref UIRR-6.
+* Display Patron's permissions and notes for the reading room. Refs UIRR-7.
+* Display patron details on patron's barcode Scan. Refs UIRR-6.
```

Comment:
Is it expected that you add changes at the top of the list?
Usually we add all changes to the bottom of the list.